package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+ <name>face_classification</name>
+ <version>1.0.0</version>
+ <description>
+ TODO!
```

Comment:
Add a small README file, outside of ros directory. With dependencies and quick installation instructions and how to run.

Reply:
Unrelated to this file but the comment is relevant.